EINVALIDTAGNAME error after v1 update

### Describe the bug

After the latest v.1 release, I have started getting this error whenever I try to install the package:

```
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "=>3.3.0" of package "vue@=>3.3.0": Tags may not have any characters that encodeURIComponent encodes.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ymans\AppData\Local\npm-cache\_logs\2023-06-21T12_45_47_775Z-debug-0.log
```

This is the complete log:
```
0 verbose cli C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js
1 info using npm@9.5.0
2 info using node@v18.15.0
3 timing npm:load:whichnode Completed in 1ms
4 timing config:load:defaults Completed in 1ms
5 timing config:load:file:C:\Program Files\nodejs\node_modules\npm\npmrc Completed in 1ms
6 timing config:load:builtin Completed in 1ms
7 timing config:load:cli Completed in 1ms
8 timing config:load:env Completed in 0ms
9 timing config:load:file:C:\Repos\MissiveNotion\.npmrc Completed in 0ms
10 timing config:load:project Completed in 1ms
11 timing config:load:file:C:\Users\ymans\.npmrc Completed in 0ms
12 timing config:load:user Completed in 1ms
13 timing config:load:file:C:\Users\ymans\AppData\Roaming\npm\etc\npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:setEnvs Completed in 0ms
16 timing config:load Completed in 6ms
17 timing npm:load:configload Completed in 6ms
18 timing npm:load:mkdirpcache Completed in 0ms
19 timing npm:load:mkdirplogs Completed in 0ms
20 verbose title npm i
21 verbose argv "i"
22 timing npm:load:setTitle Completed in 1ms
23 timing config:load:flatten Completed in 2ms
24 timing npm:load:display Completed in 2ms
25 verbose logfile logs-max:10 dir:C:\Users\ymans\AppData\Local\npm-cache\_logs\2023-06-21T12_50_21_617Z-
26 verbose logfile C:\Users\ymans\AppData\Local\npm-cache\_logs\2023-06-21T12_50_21_617Z-debug-0.log
27 timing npm:load:logFile Completed in 4ms
28 timing npm:load:timers Completed in 1ms
29 timing npm:load:configScope Completed in 0ms
30 timing npm:load Completed in 15ms
31 timing arborist:ctor Completed in 0ms
32 silly logfile start cleaning logs, removing 2 files
33 silly logfile done cleaning log files
34 timing idealTree:init Completed in 423ms
35 timing idealTree:userRequests Completed in 0ms
36 silly idealTree buildDeps
37 silly fetch manifest @huntersofbook/naive-ui-nuxt@1.0.0
38 timing arborist:ctor Completed in 0ms
39 http fetch GET 200 https://registry.npmjs.org/@huntersofbook%2fnaive-ui-nuxt 1925ms (cache revalidated)
40 timing idealTree Completed in 2354ms
41 timing command:i Completed in 2359ms
42 verbose stack Error: Invalid tag name "=>3.3.0" of package "vue@=>3.3.0": Tags may not have any characters that encodeURIComponent encodes.
42 verbose stack     at invalidTagName (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-package-arg\lib\npa.js:118:15)
42 verbose stack     at fromRegistry (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-package-arg\lib\npa.js:426:13)
42 verbose stack     at Function.resolve (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-package-arg\lib\npa.js:87:12)
42 verbose stack     at [nodeFromEdge] (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:1058:22)
42 verbose stack     at [loadPeerSet] (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:1315:36)
42 verbose stack     at [nodeFromEdge] (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:1101:30)
42 verbose stack     at async [buildDepStep] (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:929:11)
42 verbose stack     at async Arborist.buildIdealTree (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\build-ideal-tree.js:207:7)
42 verbose stack     at async Promise.all (index 1)
42 verbose stack     at async Arborist.reify (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\arborist\lib\arborist\reify.js:159:5)
43 verbose cwd C:\Repos\MissiveNotion
44 verbose Windows_NT 10.0.22621
45 verbose node v18.15.0
46 verbose npm  v9.5.0
47 error code EINVALIDTAGNAME
48 error Invalid tag name "=>3.3.0" of package "vue@=>3.3.0": Tags may not have any characters that encodeURIComponent encodes.
49 verbose exit 1
50 timing npm Completed in 2390ms
51 verbose unfinished npm timer reify 1687351821646
52 verbose unfinished npm timer reify:loadTrees 1687351821649
53 verbose unfinished npm timer idealTree:buildDeps 1687351822072
54 verbose unfinished npm timer idealTree:#root 1687351822073
55 verbose code 1
56 error A complete log of this run can be found in:
56 error     C:\Users\ymans\AppData\Local\npm-cache\_logs\2023-06-21T12_50_21_617Z-debug-0.log
```

### Reproduction

Create a project, try to add the package to it.

### System Info

```Shell
System:
    OS: Windows 10 10.0.22621
    CPU: (12) x64 12th Gen Intel(R) Core(TM) i5-12400F
    Memory: 4.37 GB / 15.82 GB
  Binaries:
    Node: 18.15.0 - C:\Program Files\nodejs\node.EXE
    npm: 9.5.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.1848.0), Chromium (114.0.1823.51)
    Internet Explorer: 11.0.22621.1
```


### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/productdevbook/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "@huntersofbook/naive-ui-nuxt",
   "type": "module",
   "version": "1.0.0",
-  "packageManager": "pnpm@8.3.0",
+  "packageManager": "pnpm@8.6.6",
   "description": "Nuxt 3 Naive UI library",
   "author": "Mehmet - productdevbook <hi@productdevbook.com>",
   "license": "MIT",
@@ -52,7 +52,7 @@
     "coverage": "vitest run --coverage"
   },
   "peerDependencies": {
-    "vue": "=>3.3.0"
+    "vue": ">=3.3.0"
   },
   "dependencies": {
     "@css-render/vue3-ssr": "^0.15.12",

